models/_sector/alm/ethereum/alm_ethereum_base_trades.sql

```diff
@@ -0,0 +1,62 @@
+{{ config(
+    schema = 'alm_ethereum'
+    , alias = 'base_trades'
+    , materialized = 'incremental'
+    )
+}}
```

Comment:
For incremental views we may need to add `unique_key` in the config blocks:
```suggestion
{{ config(
    schema = 'alm_ethereum'
    , alias = 'base_trades'
    , materialized = 'incremental'
    , incremental_strategy = 'merge'
    , unique_key = ['blockchain', 'project', 'version', 'tx_hash', 'evt_index', 'vault_address']
    )
}}
```